cluster-autoscaler/PARAMS.md

```diff
@@ -0,0 +1,117 @@
+# Full list of parameters
+
+You can grab this list by running:
+`docker run -it gcr.io/google_containers/cluster-autoscaler ./cluster-autoscaler --help`   
+
+```
+Usage of ./cluster-autoscaler:
+      --address string                                                     The address to expose prometheus metrics. (default ":8085")
```

Comment:
Please remove the leading indent and reduce the space between the flag and the description (don't align everything to the longest line)

Reply:
ping

Reply:
Hi, working on this. I'll have an updated commit later today.